src/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Todo List</title>
+    <script
+      src="https://kit.fontawesome.com/1a3881f3c5.js"
+      crossorigin="anonymous"
+    ></script>
+</head>
+<body>
+  <div class="container">
```

Comment:
Maybe it will be better to use HTML syntax instead of using only div. For example use header with class "container".